components/brave_wallet_ui/page/screens/swap/components/swap/max_slippage/max_slippage.tsx

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+import * as React from 'react'
+import Icon from '@brave/leo/react/icon'
+
+// Selectors
+import {
+  useSafeUISelector //
+} from '../../../../../../common/hooks/use-safe-selector'
+import { UISelectors } from '../../../../../../common/selectors'
+
+// Utils
+import { getLocale } from '../../../../../../../common/locale'
+
+// Styles
+import { OptionButton, CustomInput, RadioIcon } from './max_slippage.style'
+import {
+  Column,
+  LeoSquaredButton,
+  Row,
+  Text,
+  VerticalSpace
+} from '../../../../../../components/shared/style'
+
+const suggestedSlippage = '0.5'
+const mainstreamAssetPairsSlippage = '0.5'
+const stablecoinPairsSlippage = '0.5'
+const othersSlippage = '1'
```

Comment:
nit: can we use UPPER_SNAKE_CASE to help convey that these are constants?

Reply:
Changed to `UPPER_SNAKE_CASE`!